.gitattributes

```diff
@@ -0,0 +1 @@
+* eol=lf
```

Comment:
Good idea. Never saw your used syntax though. 
What about extending it directly with a charset as well?

```
root = true

[*]
end_of_line = lf
charset = utf-8
```

Reply:
Uups, forget it. Was looking at `.editorconfig` and not `.gitattributes` 😊